picard/formats/__init__.py

```diff
@@ -40,6 +40,11 @@ def supported_formats():
     return formats
 
 
+def supported_extensions():
+    extensions = [ext for exts, name in supported_formats() for ext in exts]
+    return extensions
```

Comment:
`EXTENSIONS` is a list, not a single string, because some formats support more than one extension.